packages/datadog-instrumentations/src/google-cloud-pubsub.js

```diff
@@ -0,0 +1,131 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const requestStartCh = channel('apm:google-cloud-pubsub:request:start')
+const requestAddCh = channel(`apm:google-cloud-pubsub:request:add`)
```

Comment:
This could use a better name. I had to read the subscriber to understand what it does.